akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -474,7 +596,23 @@ object ClusterReceptionistSettings {
     new ClusterReceptionistSettings(
       role = roleOption(config.getString("role")),
       numberOfContacts = config.getInt("number-of-contacts"),
-      responseTunnelReceiveTimeout = config.getDuration("response-tunnel-receive-timeout", MILLISECONDS).millis)
+      responseTunnelReceiveTimeout = config.getDuration("response-tunnel-receive-timeout", MILLISECONDS).millis,
+      // TODO: The following shouldn't require being conditional from 2.5 onward. Done in order to maintain compat.
+      heartbeatInterval =
+        if (config.hasPath("heartbeat-interval"))
+          config.getDuration("heartbeat-interval", MILLISECONDS).millis
+        else
+          2.seconds,
+      acceptableHeartbeatPause =
+        if (config.hasPath("acceptable-heartbeat-pause"))
+          config.getDuration("acceptable-heartbeat-pause", MILLISECONDS).millis
+        else
+          13.seconds,
+      failureDetectionInterval =
+        if (config.hasPath("failure-detection-interval"))
+          config.getDuration("failure-detection-interval", MILLISECONDS).millis
+        else
+          1.seconds)
```

Comment:
Ah good point, we were testing in ConductR without the reference.conf... I shall revert.